### PR TITLE
Xliff 1.2 serializer

### PR DESCRIPTION
**Pull request by dragosv**
Xliff 1.2 Serializer that is meant to be extended using the apply_xslt plugin (https://github.com/evernote/serge/pull/77)

### CODE REVIEW

**Comment by dragosv** [comment]
In order to understand the way the xliff serializer (apply_xslt on before_deserialize_ts_file and after_serialize_ts_file) take a look at https://github.com/dragosv/serge/tree/xliff_full (tests on apply_xslt). Will pull request this branch as well, once both dependent pull request are merged.

**Comment by dragosv** [comment]
To be more explicit xliff support differs between providers, requiring various changes. The two big variations are either needing resname on the trans-unit element or only supporting the id attribute and just the very basic xliff implementation (just source, target and note elements).

File: Build.PL [hunk 1]
```diff
@@ -31,6 +31,7 @@ my $builder = Module::Build->new(
         'XML::Parser'        => 0,
         'XML::Parser::Style::IxTree' => 0,
         'XML::Twig'          => 0,
+        'XML::Tidy'          => 0,
```
Comment (iafan) [inline Build.PL]: Let's not require a plugin-specific dependency at installation time. See how you can gracefully fail if the module is not present:
https://github.com/evernote/serge/blob/master/lib/Serge/Engine/Plugin/parse_csv.pm#L48-L49

File: Build.PL [hunk 2]
```diff
@@ -1,80 +1,81 @@
-use 5.010_001;
```
Comment (iafan) [inline Build.PL]: Looks like you changed the entire file from CRLF to LF. While this might be a welcome change by itself, it's not a part of the PR.

**Pull request merged**